### PR TITLE
Migrate v2.3 docs for help, console, and version to ronn-based ones

### LINKED ISSUE #830: meta: Use h1 in command references
- [x] `man`
   - #823
- [x] #839
- [x] #838
- [x] #837
- [x] #836
- [x] #835
- [x] #834
- [x] #833
- [x] #832
- [x] #831
- [x] #919
- [ ] #917

Relates to #723

### PR DESCRIPTION
**Pull request by tnir**
### What was the end-user problem that led to this PR?

`bundle_X` files are orphan from the unified man pages from rubygems/rubygems/bundler.

- Part of #723
- Closes #830

### What was your diagnosis of the problem?

We can migrate them with the one generated from ronn files.

### What is your fix for the problem, implemented in this PR?

Integrate bundle_X with man from rubygems/rubygems/bundler **only in v2.3**. Ones in other versions will be addressed in #922.

### Why did you choose this fix out of the possible options?

n/a

Signed-off-by: Takuya Noguchi [takninnovationresearch@gmail.com](https://github.com/sponsors/tnir)

### CODE REVIEW

File: source/v1.12/bundle_console.html.md [hunk 1]
```diff
@@ -2,15 +2,8 @@
 title: bundle console
 description: Start an interactive Ruby console session in the context of the current bundle
 ---
+# bundle console
 
-## bundle console
+Manual of bundle-console for Bundler 1.12 is in preparation. See the latest version of bundle-console(1) until then.
```
Comment (deivid-rodriguez) [inline source/v1.12/bundle_console.html.md:7]: What's the plan here exactly?

Comment (tnir) [reply source/v1.12/bundle_console.html.md:7]: No idea for such older versions. We can leave these as is instead.

Comment (deivid-rodriguez) [reply source/v1.12/bundle_console.html.md:7]: Yes, that was my thinking. Leave these as they are now.

Comment (tnir) [reply source/v1.12/bundle_console.html.md]: Thank you. Fixed it in f732a3f.

Comment (tnir) [reply source/v1.12/bundle_console.html.md]: And the remaining problems would be resolved in #922.

**Review by deivid-rodriguez — APPROVED** [review]
👏

**Pull request merged**